### PR TITLE
Minor fixes to mobile SDK pages

### PR DESCRIPTION
**Pull request by alexandratran**
Minor fixes and additions to the Android and iOS SDK pages.

### CODE REVIEW

**Review by elefantel — APPROVED** [review]
Looks good

File: wallet/how-to/connect/set-up-sdk/mobile/ios.md [hunk 1]
```diff
@@ -16,12 +16,39 @@ iOS SDK GitHub repository for advanced use cases.
 
 ## Prerequisites
 
-An iOS project set up with iOS version 14+.
+- MetaMask Mobile version 7.6.0 or above installed on your target device (that is, a physical device
```
Comment (bgravenorst) [inline wallet/how-to/connect/set-up-sdk/mobile/ios.md:19]: nit: It seems like we use "or later", "or above", or "Version >number>+ in the docs. I think MS guide prefers "later".

File: wallet/how-to/connect/set-up-sdk/mobile/ios.md [hunk 2]
```diff
@@ -16,12 +16,39 @@ iOS SDK GitHub repository for advanced use cases.
...
+  or emulator).
+  You can install MetaMask Mobile from the [App Store](https://apps.apple.com/us/app/metamask-blockchain-wallet/id1438144202)
+  or clone and compile MetaMask Mobile from [source](https://github.com/MetaMask/metamask-mobile)
+  and build to your target device.
+
+- iOS version 14 or above.
+
+  :::note
```
Comment (bgravenorst) [inline wallet/how-to/connect/set-up-sdk/mobile/ios.md:27]: Is this important info? Probably move out of the note then.

**Review by bgravenorst — APPROVED** [review]
Minor comments

**Pull request merged**